renovate.json

```diff
@@ -2,9 +2,43 @@
   "extends": [
     "config:base"
   ],
-  "schedule": "after 3am on Tuesday",
+  "schedule": "after 3am on Wednesday",
+  "ignorePaths": [],
   "enabledManagers": ["cargo", "npm"],
   "cargo": {
     "enabled": true
-  }
+  },
+  // groups all examples into their own single PR
```

Comment:
Comments are NOT allowed in JSON.
If you need a comment, then this is the only spec-compliant way:
```
"__next_object_comment": "groups all examples into their own single PR",
```

The other ones the same.